holoviews/core/pprint.py

```diff
@@ -363,23 +379,17 @@ def adjointlayout_info(cls, node, siblings, level, value_dims):
 
     @classmethod
     def ndmapping_info(cls, node, siblings, level, value_dims):
-
         key_dim_info = '[%s]' % ','.join(d.name for d in node.kdims)
         first_line = cls.component_type(node) + cls.tab + key_dim_info
         lines = [(level, first_line)]
+        opts = cls.option_info(node)
 
         additional_lines = []
         if len(node.data) == 0:
             return level, lines
         # .last has different semantics for GridSpace
         last = list(node.data.values())[-1]
-        if hasattr(last, 'children'):
-            additional_lines = cls.recurse(last, level=level)
-        # NdOverlays, GridSpace, Ndlayouts
-        elif last is not None and getattr(last, '_deep_indexable'):
-            level, additional_lines = cls.ndmapping_info(last, [], level, value_dims)
-        else:
-            _, additional_lines = cls.element_info(last, siblings, level, value_dims)
+        additional_lines = cls.recurse(last, level=level, value_dims=value_dims)
```

Comment:
Need to double check this, may have to re-add:

```suggestion
        if last is not None and getattr(last, '_deep_indexable'):	
             level, additional_lines = cls.ndmapping_info(last, [], level, value_dims)
        else:
            additional_lines = cls.recurse(last, level=level, value_dims=value_dims)
```